From 1ca0f5e61fe00e4e9c1dcfe13115aa0c556c1b56 Mon Sep 17 00:00:00 2001
From: Vlad Pelin <vlad.pelin@nxp.com>
Date: Mon, 31 Oct 2022 17:34:23 +0200
Subject: [PATCH 2/2] pkcs: fix wrong jump label on error case

changed a jump label on error case in pkcs-keyop

Issue: ALB-9484
Upstream-Status: Pending 

Signed-off-by: Vlad Pelin <vlad.pelin@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 examples/pkcs-keyop.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/pkcs-keyop.c b/examples/pkcs-keyop.c
index 7b71988..0f392fd 100644
--- a/examples/pkcs-keyop.c
+++ b/examples/pkcs-keyop.c
@@ -59,7 +59,7 @@ int main(int argc, char *argv[])
 	/* check how many keys are stored */
 	if (PKCS11_enumerate_keys(slot->token, &keyp, &keycount)) {
 		printf("ERROR: could not enumerate keys\n");
-		goto keygen_err;
+		goto token_err;
 	}
 	printf("Keys available: %d\n\n", keycount);
 
-- 
2.17.1

